server_installation/topics/profiles.adoc

```diff
@@ -60,6 +60,11 @@ endif::[]
 |Upload scripts through the {project_name} REST API
 |No
 |Deprecated
+
```

Comment:
s it possible to change this part based on whether we use project (enabled by default) or product (preview) ?